src/encoded/static/components/cart/util.js

```diff
@@ -8,7 +8,7 @@ import { Modal, ModalHeader, ModalBody, ModalFooter } from '../../libs/bootstrap
 
 /** List of object @type allowed in the cart. */
 const allowedCartTypes = [
-    'Experiment',
+    'Patient',
```

Comment:
Need to add Biospecimen to link to openspecimen

Maybe all Searchable objects:
Surgery
PathReport
Biospecimen